packages/teams/src/remove-team.ts

```diff
@@ -0,0 +1,23 @@
+import { UserSession } from "@esri/arcgis-rest-auth";
+import { removeGroup, unprotectGroup } from "@esri/arcgis-rest-portal";
+
+/**
+ * removeTeam unprotects a group, then deletes it
+ *
+ * @export
+ * @param {string} id Team Id
+ * @param {UserSession} authentication authentication
+ * @return {*}  {Promise<any>}
+ */
+export async function removeTeam(
+  id: string,
+  authentication: UserSession
+): Promise<any> {
+  // unprotect the group
+  const unprotectResult = await unprotectGroup({ id, authentication });
```

Comment:
This seems dangerous - I suggest we reconsider automatically protecting / unprotecting groups and leave that as a choice to the group manager

Reply:
This is a long-standing intentional pattern in Hub to limit unintended item/group deletion in the Home App. Within the Hub UI we typically have a confirmation dialog, and then remove an Item (site etc) along with it's related items/groups. If we want to change this pattern, so it's all user-controller, that's fine, but we need to accommodate that in the UI before we just change these functions.